src/v2/Apps/Settings/Routes/EditSettings/SettingsEditSettingsInformation.tsx

```diff
@@ -25,149 +25,115 @@ export const SettingsEditSettingsInformation: React.FC<SettingsEditSettingsInfor
+          try {
+            await submitUpdateSettingsInformation({
+              email,
+              name,
+              password,
+              phone,
+            })
+
+            sendToast({
+              variant: "success",
+              message: "Information updated successfully",
+            })
+          } catch (err) {
+            console.error(err)
+
+            sendToast({
+              variant: "error",
+              message: "There was a problem",
+              description: err.message,
+            })
+          }
         }}
-        validationSchema={ChangeUserInformationValidator}
-        onSubmit={onSubmit}
-        validateOnBlur={false}
       >
-        {({
-          errors,
-          handleBlur,
-          handleChange,
-          handleSubmit,
-          isSubmitting,
-          status,
-          touched,
-          values,
-        }) => (
-          <Form onSubmit={handleSubmit}>
+        {({ errors, handleBlur, handleChange, isSubmitting, values }) => (
+          <Form>
             <Join separator={<Spacer mt={2} />}>
               <Input
-                title="Full name"
+                title="Full Name"
                 name="name"
-                error={errors.name as any}
+                error={errors.name}
                 placeholder="Enter your full name"
                 value={values.name}
                 onChange={handleChange}
                 onBlur={handleBlur}
+                required
+                autoComplete="name"
               />
 
               <Input
                 title="Email"
                 name="email"
-                error={errors.email as any}
+                error={errors.email}
                 placeholder="Enter your email address"
                 value={values.email}
                 onChange={handleChange}
                 onBlur={handleBlur}
+                type="email"
+                required
+                autoComplete="email"
               />
 
               <Input
-                title="Mobile number"
+                title="Mobile Number"
                 name="phone"
                 placeholder="Enter your mobile phone number"
                 value={values.phone}
                 onChange={handleChange}
                 onBlur={handleBlur}
                 type="tel"
+                autoComplete="tel"
               />
 
               {me.paddleNumber && (
-                <>
-                  <Input
-                    name="paddleNumber"
-                    title="Bidder number"
-                    value={me.paddleNumber}
-                    readOnly
-                  />
-                </>
+                <Input
+                  name="paddleNumber"
+                  title="Bidder Number"
+                  value={"me.paddleNumber"}
+                  readOnly
+                  disabled
+                />
               )}
 
-              {touched.email && values.email !== me.email && (
```

Comment:
So you can see this was asking for `touched.email`... initially this was puzzling: apparently `touched` won't update until a blur event. So despite validation and values updating on change, you wouldn't see see this form until the blur. Simple solution: we don't need to check `touched` of course. If the value is different it's touched!

But below you can see what was happening that was letting submissions go through. If you just hit "enter" on the keyboard, it'd immediately trigger submission which would run before the input got onto the page. There was no actual validation aside from the required prop.

https://user-images.githubusercontent.com/112297/142662310-e58d9bdb-c8d0-498b-b8f2-f2a3f9bc928b.mov